chain-impl-mockchain/src/leadership/bft.rs

```diff
@@ -17,13 +16,13 @@ pub struct LeadershipData {
 
 impl LeadershipData {
     /// Create a new BFT leadership
-    pub fn new(ledger: &Ledger) -> Option<Self> {
-        if ledger.settings.bft_leaders.len() == 0 {
+    pub fn new(leaders: &Arc<[BftLeaderId]>) -> Option<Self> {
```

Comment:
nit: removing the `&` in `&Arc` would avoid double clone if the caller already had an `Arc`

Reply:
updated